src/Symfony/Component/Config/Tests/Builder/Fixtures/ScalarNormalizedTypes.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Symfony\Component\Config\Tests\Builder\Fixtures;
```

Comment:
please add the licence header to non-generated fixtures

Reply:
@nicolas-grekas Done, https://github.com/symfony/symfony/pull/46328/commits/9d16892ce773d4a26603245e13e6f6165541f181